backend/controllers/taskController.js

```diff
@@ -0,0 +1,20 @@
+const asyncHandler = require("express-async-handler");
+const Task = require("../models/userTasks")
+const bcrypt = require("bcryptjs");
+const jwt = require("jsonwebtoken");
+
+
+const createTask = asyncHandler( async (req, res) => {
+    // TODO:
+    const task = await new Task(req.body);
+    const createdTask = await task.save();
+    res.status(201).json(createdTask)
+
+    if(!createdTask) {
+        res.status(401)
```

Comment:
This should be status 400 rather than 401 since 401 will be dealt with through our authenticateToken middleware. (We'll need to later add checks to ensure that the token being provided is meant only for the correct user to use)